todo_app/templates/index.html

```diff
@@ -34,14 +34,19 @@
   </div>
 
   <div class="row">
-    <div class="col md-12">
+    <div class="col-md-12">
       <h2>Items</h2>
     </div>
   </div>
+
   <div class="row">
-    <div class="col-md-12">
+    {% for l in card_view_model.lists %} 
+    {% if l.name!="DONE" %}
+    <div class="col-md-4">
+      <h6>{{l.name}}</h6>
       {% set sort_on = request.args.sort_on|default('orderBy') %} {% for i in
```

Comment:
This logic is getting reasonably complicated - it would be clearer to move this logic into the view model as a property so that just calling the appropriate property gets the right cards, e.g. just calling `card_view_model.todo_list` and `card_view_model.doing_list` separately, or if we want to fight any duplication, we could have that be something like `card_view_model.get_ordered_cards_for_list(l.name)`

Reply:
agreed, will change refactor